ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -0,0 +1,611 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  port_channel_interfaces:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+        description:
+          type: str
+        logging:
+          type: dict
+          keys:
+            event:
+              type: dict
+              keys:
+                link_status:
+                  type: bool
+        shutdown:
+          type: bool
+        l2_mtu:
+          type: int
+          convert_types:
+          - str
+        vlans:
+          type: str
+          convert_types:
+          - int
+        snmp_trap_link_change:
+          type: bool
+        type:
+          type: str
+          valid_values:
+          - routed
+          - switched
+          - l3dot1q
+          - l2dot1q
+          description: |
+            l3dot1q and l2dot1q are used for sub-interfaces. \
+            The parent interface should be defined as routed.
+        encapsulation_dot1q_vlan:
+          type: int
```

Comment:
Please add description here and for other keys below

Reply:
schema changes updated as per comments